src/pytkdocs/objects.py

```diff
@@ -179,20 +179,24 @@ def relative_file_path(self) -> str:
 
         for namespace in namespaces:
             try:
+                importlib.import_module(namespace)
                 top_package = sys.modules[namespace]
-            except KeyError:
-                try:
-                    importlib.import_module(namespace)
-                except ImportError:
-                    return ""
-                top_package = sys.modules[namespace]
+            except (KeyError, ImportError):
+                """
+                Triggered if we can't import the package via importlib
+                Or if package isn't findable by it's canonical name even after importing
+                """
```

Comment:
Nitpicks: I'd prefer comments over a triple-quoted string :sweat_smile: : Also small typo: it's -> its